x-pack/plugins/apm/common/elasticsearch_fieldnames.ts

```diff
@@ -113,8 +113,8 @@ export const METRICSET_NAME = 'metricset.name';
 export const LABEL_NAME = 'labels.name';
 
 export const HOST = 'host';
-export const HOST_NAME = 'host.hostname';
-export const HOSTNAME = 'host.name';
+export const HOST_HOSTNAME = 'host.hostname';
```

Comment:
We might want to add a (silly) comment to this

```suggestion
export const HOST_HOSTNAME = 'host.hostname'; // Do not use. Please use `HOST_NAME` instead.
```